src/util/Constants.js

```diff
@@ -132,6 +132,10 @@ exports.Endpoints = {
         if (dynamic) format = hash.startsWith('a_') ? 'gif' : format;
         return makeImageUrl(`${root}/avatars/${userID}/${hash}`, { format, size });
       },
+      GuildAvatar: (guildId, memberId, hash, format = 'webp', size, dynamic = false) => {
```

Comment:
Since it's a GuildMember avatar, and not a GuildAvatar, this should be named GuildMemberAvatar.

Reply:
is there anything else I should rename?

Reply:
Not really, just the references in the code and typings.